<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateReminderDialog</class>
 <widget class="QDialog" name="CreateReminderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Reminder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel">
       <property name="text">
        <string>&amp;Text:</string>
       </property>
       <property name="buddy">
        <cstring>textLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="textLineEdit">
       <property name="toolTip">
        <string>Enter a description of what to be reminded</string>
       </property>
       <property name="placeholderText">
        <string>Do the thing</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="whenLabel">
       <property name="text">
        <string>&amp;When:</string>
       </property>
       <property name="buddy">
        <cstring>whenLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="whenLineEdit">
       <property name="toolTip">
        <string>Expression to describe when the reminder should be triggered</string>
       </property>
       <property name="placeholderText">
        <string>in some days at some time â€¦</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="importantLabel">
       <property name="text">
        <string>&amp;Important:</string>
       </property>
       <property name="buddy">
        <cstring>importantCheckBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="importantCheckBox">
       <property name="toolTip">
        <string>Important reminders will show up with additional visual and audio signals to be seens easier</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="helpBrowser">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="tabStopDistance">
      <double>40.000000000000000</double>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textLineEdit</tabstop>
  <tabstop>whenLineEdit</tabstop>
  <tabstop>importantCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateReminderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateReminderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
